### PR TITLE
api integration using redux tool kit

### PR DESCRIPTION
**Pull request by rahulc-kv**

### CODE REVIEW

File: src/containers/home/HomeContainer.tsx [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { useSelector } from "react-redux";
 
 import { RootState } from "../../store/reducers";
 import { useAppDispatch } from "store/store";
-import { useGetBaseQuery } from "./api";
+import { useGetCountryQuery } from "./api";
 import { updateValue } from "./slice";
 import { Button } from "components";
 
 const HomeContainer: React.FC = () => {
```
Comment (deepak-keyvalue) [inline src/containers/home/HomeContainer.tsx]: Lets simply name it Home instead of HomeContainer?

File: src/containers/home/HomeContainer.tsx [hunk 2]
```diff
@@ -3,12 +3,14 @@ import { useSelector } from "react-redux";
...
-  const { isLoading } = useGetBaseQuery();
+  const { isLoading, data: countries } = useGetCountryQuery();
+  console.log(countries, isLoading);
```
Comment (deepak-keyvalue) [inline src/containers/home/HomeContainer.tsx]: Remove logs

File: src/containers/home/HomeContainer.tsx [hunk 3]
```diff
@@ -21,6 +23,11 @@ const HomeContainer: React.FC = () => {
       <p className="mt-10">
         {data}
       </p>
+      <p className="mt-10">
+        {
```
Comment (deepak-keyvalue) [inline src/containers/home/HomeContainer.tsx:26]: See if we can change it to  { countries?.map((country) => (<div>{country.name}</div>)) }

File: src/store/store.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { configureStore } from "@reduxjs/toolkit";
 import { setupListeners } from "@reduxjs/toolkit/query";
-import { baseApi } from "containers/home/api";
+import  baseApi from "services/api";
```
Comment (deepak-keyvalue) [inline src/store/store.ts]: I think we have alias, but lets make it like @services

**Review by harikamathkeyvalue — APPROVED** [review]

File: src/containers/home/HomeContainer.tsx (hunk 1 above)
Comment (harikamathkeyvalue) [reply src/containers/home/HomeContainer.tsx]: Seeing multiple instances of Home containers arising, hence changing the container name from 'Home Container' to 'Home' and the page name from 'Home' to 'Homepage'

File: src/containers/home/HomeContainer.tsx (hunk 2 above)
Comment (harikamathkeyvalue) [reply src/containers/home/HomeContainer.tsx]: Removed

File: src/containers/home/HomeContainer.tsx (hunk 3 above)
Comment (harikamathkeyvalue) [reply src/containers/home/HomeContainer.tsx:26]: Updated

File: src/store/store.ts (hunk 4 above)
Comment (harikamathkeyvalue) [reply src/store/store.ts]: Added alias.

**Pull request merged**